mlperf_logging/compliance_checker/hpc_1.0.0/common.yaml

```diff
@@ -0,0 +1,162 @@
+    CHECK: " v['value'] != '' "
+
+- KEY:
+    NAME:  submission_platform
+    REQ:   EXACTLY_ONE
+    CHECK: " v['value'] != '' "
+
+- KEY:
+    NAME:  submission_division
+    REQ:   EXACTLY_ONE
+    CHECK: " v['value'] in ['closed', 'open'] "
+    POST:  " enqueue_config('hpc_1.0.0/{}_common.yaml'.format(v['value'])) "
+
+- KEY:
+    NAME:  submission_status
+    REQ:   EXACTLY_ONE
+    CHECK: " v['value'] in ['cloud', 'onprem', 'preview', 'research'] "
+
+# at least one record should be found, but any found records must pass the test
+- KEY:
+    NAME:  cache_clear
+    REQ:   AT_LEAST_ONE
+    CHECK:
+        - "'value' in v"
+
+# frequency not checked
+- KEY:
+    NAME:  init_start
+    REQ:   AT_LEAST_ONE
+    CHECK:
+        - "not s['init_stopped']"
+        - "not s['run_started']"
+    POST:  " s['init_started'] = True; s['first_init_start']=min(s['first_init_start'], ll.timestamp) "
+
+# confirm less than 20min since the very first init_start
+- KEY:
+    NAME:  init_stop
+    REQ:   EXACTLY_ONE
+    CHECK:
+        - "s['init_started']"
+        - "not s['run_started']"
+        - "ll.timestamp - s['first_init_start'] < (30*60*1e3)"
+    POST:  " s['init_stopped'] = True"
+
+# HPC requires data staging to be included in run timing
+- KEY:
+    NAME: staging_start
+    CHECK:
+        - "s['run_started']"
+    POST: " s['staging_started'] = True "
+
+- KEY:
+    NAME: staging_stop
+    CHECK:
+        - "s['staging_started']"
+
+# run start and run stop
+- KEY:
+    NAME:  run_start
+    REQ:   EXACTLY_ONE
+    CHECK: " ( s['init_stopped'] == True )"
+    POST:  " s['run_started'] = True "
+
+# status can also be aborted, but not allowing it here for now
+# if eval is inside epoch and we decide to terminate, we can lack epoch_stop, it is ok
+- KEY:
+    NAME:  run_stop
+    REQ:   EXACTLY_ONE
+    CHECK:
+        - "s['run_started']"
+        - "'status' in v['metadata']"
+    POST:  " s['run_stopped'] = True "
+
+# FIXME: check epoch_count value match
+- KEY:
+    NAME:  block_start
+    REQ:   AT_LEAST_ONE_OR(epoch_start)
+    CHECK:
+        - "s['run_started']"
+        - "'epoch_count' in v['metadata']"
+        - "'first_epoch_num' in v['metadata']"
+        - "v['metadata']['epoch_count'] > 0"
+
+- KEY:
+    NAME:  block_stop
+    REQ:   AT_LEAST_ONE_OR(epoch_stop)
+    CHECK:
+        - "'first_epoch_num' in v['metadata']"
+
+- KEY:
+    NAME:  epoch_start
+    REQ:   AT_LEAST_ONE_OR(block_start)
+    CHECK:
+        - "'epoch_num' in v['metadata']"
+
+- KEY:
+    NAME:  epoch_stop
+    REQ:   AT_LEAST_ONE_OR(block_stop)
+    CHECK:
+        - "'epoch_num' in v['metadata']"
+
+# making sure previous eval did print it's accuracy result
+- KEY:
+    NAME:  eval_start
+    REQ:   AT_LEAST_ONE_OR(block_start)
+    CHECK:
+        - "'epoch_num' in v['metadata']"
+
+- KEY:
+    NAME:  eval_stop
+    REQ:   AT_LEAST_ONE_OR(block_stop)
+    CHECK:
+        - "'epoch_num' in v['metadata']"
+
+- KEY:
+    NAME:  eval_accuracy
```

Comment:
we use eval_error for the other two benchmarks, so this has to be removed from common